src/test/regress/expected/multi_mx_router_planner.out

```diff
@@ -456,15 +456,6 @@ DEBUG:  Plan is router executable
 SELECT * FROM articles_hash_mx, position('om' in 'Thomas') WHERE author_id = 1 or author_id = 2;
 ERROR:  could not run distributed query with complex table expressions
 HINT:  Consider using an equality filter on the distributed table's partition column.
--- subqueries are not supported in WHERE clause in Citus
```

Comment:
We have a bunch of tests that covers these cases. I really don't see value in keeping these tests. Do you?